[feat] atulização cadastre e rotas

germannafol opened the pull request:
POR FAVOR, EDITE ESSA MENSAGEM INSERINDO AS INFORMAÇÕES DO SEU PROJETO. COMECE APAGANDO ESSA LINHA.

### INSIRA O NOME NOME DO PROJETO AQUI

### O que funciona
- DESCREVA O QUE FUNCIONA NO SEU PROJETO

### O que não funciona
- DESCREVA O QUE NÃO FUNCIONA NO SEU PROJETO

### Link Surge 
A PARTIR DA SEMANA 5, INSIRA AQUI O LINK DO SURGE. **ANTES DISSO, APAGUE ESSAS DUAS LINHAS.**

### Imagens
TIRE PRINTS DAS TELAS DO SEU SITE E COLE AQUI

germannafol committed:
[feat] atulização cadastre e rotas

germannafol committed:
[feat] atulização do cadastro

germannafol committed:
[feat] atualização

germannafol committed:
Atualizações das funções onChange

Wanuzia reviewed (changes requested):
Ser você subir esse PR, vai estar criando outra pasta para cadastro, mas com nome de FeedCadastre.

Sugiro atualizar a branch, pra não ter duplicações. Na branch master atual, tem a pasta SignUp, que tem estes arquivos já atualizados.

germannafol committed:
Atualização Signup

germannafol commented:
Eu arrumei

germannafol committed:
Add do Head

Wanuzia commented:
Show!😁

Wanuzia committed:
Update Router.js

Voltei a props para o Router, pra não quer a rota de ir para cadastro. Não sei se essa props está te atrapalhando, se estiver, só comentar na hora de rodar o código e me informar, pra eu encontrar outra forma para os testes passarem.

Wanuzia committed:
Update Cadastre.js

 a tag Subtext é um parágrafo e dentro dela estava outro parágrafo. Isso estava dando erro no console.
